# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/sysklogd/arch-alpha-ksymmod.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./ksym_mod.c.orig	Sun Oct  8 00:10:58 2000
+++ ./ksym_mod.c	Sun Oct  8 00:12:26 2000
@@ -80,6 +80,8 @@
  *	Changed llseek() to lseek64() in order to skip a libc warning.
  */
 
+/* On Alpha system this header file produces a compiler error */
+#define _ALPHA_CURRENT_H   // disable asm/current.h [2.4.0-test9]
 
 /* Includes. */
 #include <stdlib.h>
